apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -42,12 +38,55 @@ TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a t
 ## Without branding
 
 ```jsx live
-<VideoPlayer branding={false} title="GitHub media player">
+<VideoPlayer hasBranding={false} title="GitHub media player">
+  <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## With visually hidden title
+
+```jsx live
+<VideoPlayer visuallyHiddenTitle title="GitHub media player">
   <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
   <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
 </VideoPlayer>
 ```
 
+## With custom controls
+
+```jsx live
+<VideoPlayer
+  title="GitHub media player"
+  showPlayPauseButton={false}
+  showSeekControl={true}
+  showCCButton={false}
+  showMuteButton={false}
+  showVolumeControl={false}
+  showFullScreenButton={false}
+>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## Hide controls when paused
+
+```jsx live
+<VideoPlayer title="GitHub media player" showControlsWhenPaused={false}>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## Minimal
+
```

Comment:
Missing the example for this one...

![Screenshot 2024-07-17 at 15 04 54](https://github.com/user-attachments/assets/324cc68d-9ab0-452c-bfdf-e6713e613fe4)

Reply:
Thanks!

44b758df161f2d3883c39cf3e20df8d8064dd45a